index.md

```diff
@@ -4,19 +4,17 @@ layout: page
 
 # Free PDK Documentation
 
-*Free PDK* is an effort to create an open source alternative to the proprietary
-[Padauk](http://www.padauk.com.tw/index_en.aspx) µC programmer,
-as well as adding support to [SDCC](http://sdcc.sourceforge.net/) for Padauk µCs.
+*Free PDK* is an open sourced and independently created tool-chain for the [Padauk](http://www.padauk.com.tw/index_en.aspx) 8-Bit Microcontrollers, created as an alternative to the proprietary and closed tools provided by the Taiwanese company itself.
+
+This includes the EasyPDKProg µC programmer hardware, adding support to the [SDCC](http://sdcc.sourceforge.net/) C-Compiler for Padauk µC, as well
```

Comment:
```suggestion
This includes the EasyPDKProg µC programmer hardware, adding support to the [SDCC](http://sdcc.sourceforge.net/) C-Compiler for Padauk µCs, as well
```

Reply:
Looks good otherwise :+1:

Reply:
True! See update. The sentence order was off, anyways.